docs/manual/scala/guide/build/ScalaComponents.md

```diff
@@ -0,0 +1,96 @@
+# Scala Components
+
+The Lagom scaladsl requires building an Application that mixes in all the `Components` that will enable the required features for your code. This means each `Application` will need a specific set of cake slices. Not only that but `Dev` mode and `Prod` mode need adjusting since the environments where the Service is run vary.
```

Comment:
Hmmm, I was on cruise control and thought we had that explained in `Implementing Services` already. Just checked and that section is missing a link or an explanation too. 

This page looks like an appropriate location for an intro to `cake`.